packages/api/src/feature_flag/service/create.test.ts

```diff
@@ -0,0 +1,189 @@
+import request from "supertest";
```

Comment:
Since we're testing at the controller level, this file should be at /src/feature_flag/controller.test.ts